cram_common/cram_manipulation_interfaces/src/trajectories.lisp

```diff
@@ -83,19 +83,19 @@
                    `(and (cram-robot-interfaces:robot ?robot)
                          (cram-robot-interfaces:standard-to-particular-gripper-transform
                           ?robot ?transform))))))))
-  (lambda (object-to-standard-gripper)
-    (when object-to-standard-gripper
-      (let ((base-to-standard-gripper-transform
-              (cram-tf:multiply-transform-stampeds
-               cram-tf:*robot-base-frame* gripper-tool-frame
-               object-transform          ; bTo
-               object-to-standard-gripper ; oTg'
-               :result-as-pose-or-transform :transform))) ; bTo * oTg' = bTg'
-        (cram-tf:multiply-transform-stampeds ; bTg' * g'Tg = bTg
-         cram-tf:*robot-base-frame* gripper-tool-frame
-         base-to-standard-gripper-transform      ; bTg'
-         standard-to-particular-gripper-transform ; g'Tg
-         :result-as-pose-or-transform :pose))))))
+    (lambda (object-to-standard-gripper)
+      (when object-to-standard-gripper
+        (let ((base-to-standard-gripper-transform
+                (cram-tf:multiply-transform-stampeds
+                 cram-tf:*robot-base-frame* gripper-tool-frame
+                 object-transform          ; bTo
+                 object-to-standard-gripper ; oTg'
+                 :result-as-pose-or-transform :transform))) ; bTo * oTg' = bTg'
+          (cram-tf:multiply-transform-stampeds ; bTg' * g'Tg = bTg
+           cram-tf:*robot-base-frame* gripper-tool-frame
+           base-to-standard-gripper-transform      ; bTg'
+           standard-to-particular-gripper-transform ; g'Tg
+           :result-as-pose-or-transform :pose))))))
```

Comment:
Please don't commit things like this. They create unnecessary merge conflicts just because your emacs has a different preference for indentation than mine.

Reply:
This relates more to common packages that multiple people might be working on at the same time. In your packages fill free to fix the indentation but in common packages try to refrain from committing indentation fixes.